tests/Tests.elm

```diff
@@ -221,11 +221,23 @@ tests =
                   Expect.equal
                     (List.map ((+) 1) xs |> Stack.fromList)
                     (Stack.fromList xs |> Stack.map ((+) 1))
-          
-          , test "mapping an empty stack should result in an empty stack" <| 
+
+          , test "mapping an empty stack should result in an empty stack" <|
               \() ->
                   Expect.equal
                     Stack.empty
                     (Stack.map (always "foobar") Stack.empty)
           ]
-    ]
\ No newline at end of file
+    , describe "filter"
```

Comment:
Thanks for adding these! I know that this is might be overkill, but let's add a test for filtering an empty stack, should also be an empty stack.